### PR TITLE
Sync Streams & typos

### PR DESCRIPTION
**Pull request by teoreteetik**
This adds snippets for the new Sync Streams API (all languages) and fixes some unrelated typos.
The CI failures seem to be unrelated to these changes, but lmk if I can fix those somehow.

### CODE REVIEW

File: sync/rest/streams/list-streams/meta.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "title": "Retrieve a list of Sync Streams",
```
Comment (dr0nius) [inline sync/rest/streams/list-streams/meta.json]: Message Streams

File: sync/rest/streams/create-stream/meta.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "title": "Create a Sync Stream",
```
Comment (dr0nius) [inline sync/rest/streams/create-stream/meta.json]: Message Stream

File: sync/rest/streams/delete-stream/meta.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "title": "Delete a Sync Stream",
```
Comment (dr0nius) [inline sync/rest/streams/delete-stream/meta.json]: Message Stream

File: sync/rest/streams/retrieve-stream/meta.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+  "title": "Retrieve a Sync Stream",
```
Comment (dr0nius) [inline sync/rest/streams/retrieve-stream/meta.json]: Message Stream

Comment (teoreteetik) [reply sync/rest/streams/retrieve-stream/meta.json]: Fixed

File: sync/rest/streams/delete-stream/meta.json (hunk 3 above)
Comment (teoreteetik) [reply sync/rest/streams/delete-stream/meta.json]: Fixed

File: sync/rest/streams/create-stream/meta.json (hunk 2 above)
Comment (teoreteetik) [reply sync/rest/streams/create-stream/meta.json]: Fixed

File: sync/rest/streams/list-streams/meta.json (hunk 1 above)
Comment (teoreteetik) [reply sync/rest/streams/list-streams/meta.json]: Fixed

**Comment by atbaker** [comment]
Hmm. Looks like the errors are related to TwiML generation somehow. So I agree they're unrelated.

Our build has been a little wonky recently so I'll go ahead and merge this. We expect to have time after SIGNAL London to figure out what's wrong with ol' Travis.

**Pull request merged**